MapleServer2/PacketHandlers/Game/FieldEnterHandler.cs

```diff
@@ -38,14 +38,34 @@ public class FieldEnterHandler : GamePacketHandler {
             }
 
             // Add catalysts for testing
-            /*
+            
             int[] catalysts = { 40100001, 40100002, 40100003, 40100021, 40100023, 40100024, 40100026 };
-            foreach (int catalyst in catalysts) {
-                var item = new Item(catalyst) { Amount = 99999, Uid = catalyst };
+
+            //Adds first set of item test case
+            foreach(int catalyst in catalysts)
+            {
+                var item = new Item(catalyst) { Amount = 4, Uid = catalyst };
+
                 session.Inventory.Add(item);
                 session.Send(ItemInventoryPacket.Add(item));
             }
-            */
+            //session.Inventory.print();
```

Comment:
We want to get rid of test code like this after we're done testing with it, we don't want to leave commented out test code in the master branch.

Reply:
Gotcha will update.